nemo/collections/nlp/models/language_modeling/megatron_gpt_sft_model.py

```diff
@@ -400,7 +400,20 @@ def inference_step(self, dataloader_iter, batch_idx, mode, dataloader_idx=0):
         self._reconfigure_and_process_inference_batch(batch, data_cfg)
         # Meta data from dataset
         metadata = batch.get('metadata', [{}] * len(batch['tokens']))
-        loss = super().validation_step(itertools.chain([batch]), batch_idx)
```

Comment:
if the issue happens because of prefetch-ing when len(dataloader) ==1, what happens during GPT pre-training when val set is small enough to be just 1 batch?